### ISSUE TITLE
Example is not correctly selected

### ISSUE THREAD

**Issue by siritinga** [mvysny/aedict#470]
Hi.

I found a funny issue. This is how I reproduce it:

- Install the Tatoeba and the Kotozawa example dictionaries.
- Configure the GUI to show example sentences in first tab.
- Search for うすい and select the first entry, 薄い.
- Select the "KANJI" tab and tap on the kanji.

This is what I see at this point:
![2015-04-03 16 14 35](https://cloud.githubusercontent.com/assets/3969105/6984884/c012e32e-da2d-11e4-8d02-50f35e16d548.png)
- Go down to the 6th example: 建ての薄着.
- Now, if I tap on that example, the wrong example is displayed:

![2015-04-03 16 16 36](https://cloud.githubusercontent.com/assets/3969105/6984885/c016d290-da2d-11e4-84cf-f3e42f144a65.png)

It doesn't happend if the Kotowaza file is not installed or if examples are shown in its own tab. 

Can you reproduce it?

**Comment by mvysny** [comment]
Hi, many thanks for such detailed instructions. Yes, I can reproduce it as well. This bug is interesting ;) the english translation is correct, but the sentence is completely changed. This is also visible on 7th, 8th and some more sentences, I'm very curious what the nature of this bug is.

**Comment by mvysny** [comment]
Hm, it seems that there is a bug in the example sentences display code - it seems that the first kotowaza proverb (date no usugi) is repeated into the 6th sentence. So, the example sentence detail actually shows the correct sentence. I'll look more into this.

**Comment by siritinga** [comment]
I'm glad you are enjoying it, and keeping me posted. I'm also a software developer (in a very different field) and this bug looked like a funny one :)

**Comment by mvysny** [comment]
Fixed ;) This was an interesting one. There is a View (android ui component) for each *visible* (not for all rows, just for those rows which are actually shown on screen) row of the list. If a list row is scrolled out of the screen, the view is "recycled" - reused for another row which just happened to be scrolled in to the screen. Now, in my Tatoeba View I have two children views - one which can display the Furigana and other which shows kanji and reading on two lines. Only one is shown at the same time, depending on whether the furigana transcription is available for the row or not.
Now, what happened is that the first row (date no usugi) got scrolled away and recycled into the 6th row. I changed the text of the Furigana view correctly to "ano hito wa...", but I forgot to change its visibility, so the two-row View was shown instead, showing the text "date no usugi" :-D
Fixed in Aedict 3.16, should be out next week.
